bin/kafka-run-class.sh

```diff
@@ -286,7 +286,7 @@ if [ "x$GC_LOG_ENABLED" = "xtrue" ]; then
   # 10 -> java version "10" 2018-03-20
   # 10.0.1 -> java version "10.0.1" 2018-04-17
   # We need to match to the end of the line to prevent sed from printing the characters that do not match
-  JAVA_MAJOR_VERSION=$($JAVA -version 2>&1 | sed -E -n 's/.* version "([0-9]*).*$/\1/p')
+  JAVA_MAJOR_VERSION=$($JAVA -version 2>&1 | sed -n '1p'|awk -F '.' '{print $2}')
```

Comment:
This incorrectly changes behaviour on openjdk 1.8 and 9, and is returning the minor version rather than the major version.  Can you explain what you're trying to achieve here?